docs/sphinx_doc/en/source/tutorial/203-model.md

```diff
@@ -418,12 +420,38 @@ Here we provide example configurations for different model wrappers.
 #### Post Request API
 
 <details>
-<summary>Post request API (<code><a href="https://github.com/modelscope/agentscope/blob/main/src/agentscope/models/post_model.py">agentscope.models.PostAPIModelWrapperBase</a></code>)</summary>
+<summary>Post Request Chat API (<code><a href="https://github.com/modelscope/agentscope/blob/main/src/agentscope/models/post_model.py">agentscope.models.PostAPIModelWrapperBase</a></code>)</summary>
```

Comment:
please change 'agentscope.models.PostAPIModelWrapperBase' to 'agentscope.models.PostAPIChatWrapper'